### PR TITLE
Updated What's New with changes since October 1.

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

This pull request updates the [What's New](https://devdocs.magento.com/whats-new.html) page with the documentation changes published since the last update on October 1.

## Affected DevDocs pages

- https://devdocs.magento.com/whats-new.html

### CODE REVIEW

File: _data/whats-new.yml [hunk 1]
```diff
@@ -4,10 +4,202 @@ description: |
...
+  type: New topic
+  date: October 8, 2019
+  link: https://github.com/magento/devdocs/pull/4882
+- description: Added documentation for the [applyStoreCreditToCart](https://devdocs.magento.com/guides/v2.3/graphql/mutations/apply-store-credit.html)
+    and [removeStoreCreditFromCart](https://devdocs.magento.com/guides/v2.3/graphql/mutations/remove-store-credit.html) to the _GraphQL Developer Guide_.
+  versions: 2.3.3
+  type: New topic
+  date: October 8, 2019
+  link: https://github.com/magento/devdocs/pull/4960
+- description: Added documentation for the [setPaymentMethodAndPlaceOrder](https://devdocs.magento.com/guides/v2.3/graphql/reference/set-payment-place-order.html)
```
Comment (dshevtsov) [inline _data/whats-new.yml]: The link redirects to home page

File: _data/whats-new.yml [hunk 2]
```diff
@@ -4,10 +4,202 @@ description: |
...
+    to the _GraphQL Developer Guide_.
+  versions: 2.3.3
+  type: New topic
+  date: October 8, 2019
+  link: https://github.com/magento/devdocs/pull/4916
+- description: Added information about applying store codes to [asynchronous](https://devdocs.magento.com/guides/v2.3/rest/asynchronous-web-endpoints.html)
+    and [bulk](https://devdocs.magento.com/guides/v2.3/rest/bulk-endpoints.html) REST
+    endpoints in the _REST API Reference_.
+  versions: ''
```
Comment (dshevtsov) [inline _data/whats-new.yml]: Empty versions

Comment (meker12) [inline _data/whats-new.yml]:
```suggestion
  versions: '2.3.3'
```

File: _data/whats-new.yml (hunk 1 above)
Comment (meker12) [reply _data/whats-new.yml]: Removed this item per instructions from @keharper

**Comment by meker12** [comment]
running tests

**Pull request merged**